components/style/values/specified/font.rs

```diff
@@ -610,6 +611,296 @@ impl Parse for FontSize {
     }
 }
 
+bitflags! {
+    #[cfg_attr(feature = "servo", derive(MallocSizeOf))]
+    /// Flags of variant alternates in bit
+    pub struct ParsingFlags: u8 {
```

Comment:
Let's call this something more specific like `VariantAlternatesParsingFlags`.

Reply:
Also, no need for `pub` I think.

Reply:
Oh, sorry for that I forgot to remove the `pub`. I'll update it!